### DISCUSSION TITLE
Best practice for storing additional customer data in database

### DISCUSSION THREAD

**Discussion by Web10-Joris** [duncanmcclean/simple-commerce#1033] [Help] [answered]
Hi,

I'm trying to figure out the best practice for storing additional customer data when using Runway for customers and orders. For instance, we want to keep a record of the phone number and company name for each customer. I've created a migration to add these columns to the customer table and included them in Runway's customer blueprint. When I input this data via the database or control panel, everything works fine, but I can't seem to store this information using the cart update route (with frontend form tag). Could I be doing something wrong? I've already whitelisted the phone and company fields for the customer in the config.

**Comment by Web10-Joris** [comment]
Forgot to mention, when searching for this I also found out that [this related link](https://simple-commerce.duncanmcclean.com/troubleshooting#:~:text=Information%20not%20being%20saved%20onto%20orders/customers) in the docs isn't working anymore.

**Reply by duncanmcclean** [reply]
Thanks for pointing this out. I've fixed the link.

**Comment by duncanmcclean** [comment]
Can you share the code for the checkout template where customer enter their phone number & company name?

**Reply by Web10-Joris** [reply]
Ofcourse! You should still have acces to my repository. [This](https://github.com/Web10-Joris/JouwTech/blob/main/resources/views/checkout/checkout.antlers.html) is the file for the checkout.

**Reply by Web10-Joris** [reply, marked as answer]
Ah now I see the fixed page in the documentation, I'm not using the array for customers inside the input field :)
```
{{ sc:cart:update }}
    <input type="date" name="customer[dob]">
{{ /sc:cart:update }}
```

**Reply by duncanmcclean** [reply]
Ah yes, you need to do that otherwise it'll try saving to the order's data instead.
